omnibus/files/private-chef-cookbooks/private-chef/templates/default/elasticsearch.yml.erb

```diff
@@ -18,6 +18,16 @@ network.host: "<%= @listen %>"
 #            http://www.elastic.co/guide/en/elasticsearch/reference/current/modules-discovery.html
 http.port: <%= @port %>
 
+<%
+  # ES7 mandatory configuration.These settings are applicable only for the users using standalone or tiered configurations. 
+  # For BYOD(external elasticsearch) none of the omnibus configs are applicable.
+  if node['private_chef']['elasticsearch']['es_version'].to_f > 7.0
+%>
+discovery.type: single-node
```

Comment:
It might be useful to leave a note explaining that we have not enabled bootstrap checks, so failing bootstrap checks will appear as warnings in the es logs. https://www.elastic.co/guide/en/elasticsearch/reference/current/bootstrap-checks.html